src/node_i18n.h

```diff
@@ -42,23 +42,23 @@ bool InitializeICUDirectory(const std::string& path);
 
 void SetDefaultTimeZone(const char* tzid);
 
-enum idna_mode {
+enum class idna_mode {
   // Default mode for maximum compatibility.
-  IDNA_DEFAULT,
+  kDEFAULT,
```

Comment:
If you look at the naming convention of other enum class entries in the rest of the codebase, those are either `k` followed by the name in `PascalCase`, like `kDefault` or those are written in `SCREAMING_SNAKE_CASE` like `DEFAULT`. I'm okay with either of the two (slight preference for the first naming convention because the second one looks more like macros to me) as long as we don't mix both styles together, which is what's happening here.

Reply:
Thank you for detail explanation. Applied the first naming convention.